code/clue.js

```diff
@@ -1,59 +1,227 @@
 // STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+//
 
-const mrGreen = {
-  firstName: 'Jacob',
+const MrGreen = {
+  firstName : 'Jacob',
   lastName: 'Green',
   color: 'green',
   description: 'He has a lot of connections',
   age: 45,
   image: 'assets/green.png',
   occupation: 'Entrepreneur'
+}  
+console.log(MrGreen.firstName);
+
+const ProfessorPlum = {
+  firstName : 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'He works hard',
+  age: 35,
+  image: 'assets/plum.png',
+  occupation: 'Teacher'
+}
+const MissScarlet ={
+  firstName : 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She is smart',
+  age: 25,
+  image: 'assets/scarlet.png',
+  occupation: 'Broker'
 }
 
+const MrsPeacock = {
+  firstName : 'Jane',
+  lastName: 'Aka - Mrs Peacock',
+  color: 'blue',
+  description: 'Socialite',
+  age: 35,
+  image: 'assets/peacock.png',
+  occupation: 'Manager'
+} 
+const ColonelMustard = {
+  firstName : 'Jack',
+  lastName: 'Mustard',
+  color: 'Yellow',
+  description: 'Wise old man',
+  age: 75,
+  image: 'assets/mustard.png',
+  occupation: 'Retired Colonel'
+} 
+const MrsWhite = {
+  firstName : 'Mrs',
+  lastName: 'White',
+  color: 'White',
+  description: 'Mother of the house hold',
+  age: 65,
+  image: 'assets/white.png',
+  occupation: 'House keeper'
+} 
+
+
 // CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
   name: 'rope',
   weight: 10
 }
+const knife = {
+  name: `knife`,
+  weight: 30
+}
+const candlestick = {
+  name: `candlestick`,
+  weight: 5
+}
+const dumbbell = {
+  name: `bumbbell`,
+  weight: 80
+}
+const poison = {
+  name: `poison`,
+  weight: 30
+}
+const axe = {
+  name: `kaxe`,
+  weight: 15
+}
+
+const bat = {
+  name: `bat`,
+  weight: 32
+}
+const trophy = {
+  name: `trophy`,
+  weight: 18
+}
+
+const pistol = {
+  name: `pistol`,
+  weight: 55
+}
+
+console.log(rope.weight);
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
+const rooms = [
+  `Diningroom`, 
+  `Conservatory`, 
+  `Kitchen`, 
+  `Study`, 
+  `Library`, 
+  `Billiard room`, 
+  `Lounge`, 
+  `Ballroom`, 
+  `Hall`, 
+  `Spa`, 
+  `Living room`,
+  `Observatory`, 
+  `Theater`,
+  `Guest House`, 
+  `Patio`
+];
+
 // NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
 
-const suspects = [
-  mrGreen,
-  mrsWhite
-  // ...  and the rest
+let suspects = [
+  MrGreen,
+  ProfessorPlum,
+  MissScarlet,
+  MrsPeacock,
+  ColonelMustard,
+  MrsWhite
+]
```

Comment:
I'm a bit curious why you chose `let` instead of `const` here?  
Unless you expect `suspects` to change to a different value while the webpage is running this can be a constant.